### PR TITLE
Changes the Main Page to Make Upload Images and Tag Images children 

### PR DESCRIPTION
**Pull request by ManvithaPonnapati**
This prevents the issue during routing/reload.

### CODE REVIEW

File: reactapp/src/components/header.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ class Header extends React.Component {
             <nav className="navbar navbar-expand-lg navbar-dark teal mb-4 pageNavHeader" id="home">
                 <span className="navbar-brand" href="#">
                     <Link to="/">
-                        <img className="logoImg" alt="ClimateChange.AI" src={logo}/>
+                        <img className="logoImg" alt="ClimateChange.AI" src={logo} onClick={()=>{window.location.reload()}}/>
```
Comment (ManvithaPonnapati) [inline reactapp/src/components/header.js]: On clicking the logo, the page is reloaded (We are on a single page app - So reload automatically resets to the original "home" state.

File: reactapp/src/components/tagImages.js [hunk 2]
```diff
@@ -400,8 +398,7 @@ export class TagImages extends React.Component {
     }
 
     componentWillMount() {
-        if (this.props.location.state)
-            this.loadImages(this.props.location.state.files);
+
```
Comment (ManvithaPonnapati) [inline reactapp/src/components/tagImages.js]: We no longer need this

File: reactapp/src/styles/image_uploader.css [hunk 3]
```diff
@@ -25,12 +25,13 @@
...
     margin-left: auto;
     margin-right: auto;
+    margin-bottom:10px;
     text-align: center;
     color: white;
 }
 
 .main-page .up-screen {
-    min-height: 100vh;
+    min-height: 60vh;
```
Comment (ManvithaPonnapati) [inline reactapp/src/styles/image_uploader.css]: Changed the CSS to handle all the white space on the page

File: reactapp/src/components/header.js (hunk 1 above)
Comment (notoraptor) [reply reactapp/src/components/header.js]: This is already done by the `<Link>` component surrounding the `img` !

**Review by notoraptor — APPROVED** [review]
Hi @ManvithaPonnapati ! According to discussions on slack yesterday with Victor, I assumed the current format is convenient (ie. the about section being at bottom of homepage by default, and displayed only when scrolling on when clicking on navigation link; I just had to fix extra unnecessary scrolling yesterday, and it was ok).

So I just made a commit to keep current stylesheets and quickly reformat code.

**Pull request merged**